app/src/renderer/vuex/modules/user.js

```diff
@@ -98,12 +98,19 @@ export default ({ commit, node }) => {
     loadErrorCollection({ state, dispatch }, account) {
       let errorCollection =
         localStorage.getItem(`${ERROR_COLLECTION_KEY}_${account}`) === "true"
-      dispatch("setErrorCollection", { account, optin: errorCollection })
+      if (state.errorCollection !== errorCollection)
+        dispatch("setErrorCollection", { account, optin: errorCollection })
     },
-    setErrorCollection({ state }, { account, optin }) {
+    setErrorCollection({ state, commit }, { account, optin }) {
+      if (state.errorCollection !== optin && config.development) {
+        commit("notifyError", {
+          title: `Couldn't switch ${optin ? "on" : "off"} error collection`,
```

Comment:
Let's put periods at the end of sentences.

Reply:
Sounds good to me.